Project/src/test/java/com/github/lgooddatepicker/components/TestCalendarPanel.java

```diff
@@ -222,23 +220,17 @@ public void TestDateHighlightAndVetoPolicy()
           NoSuchMethodException, InvocationTargetException {
     DatePickerSettings settings = new DatePickerSettings(Locale.ENGLISH);
     settings.setHighlightPolicy(
-        new DateHighlightPolicy() {
-          @Override
-          public HighlightInformation getHighlightInformationOrNull(LocalDate date) {
-            if (date.get(ChronoField.DAY_OF_MONTH) % 2 == 0) {
-              return new HighlightInformation(Color.green, Color.blue, "highlighted");
-            }
-            return null;
+        date -> {
+          if (date.get(ChronoField.DAY_OF_MONTH) % 2 == 0) {
+            return new HighlightInformation(Color.green, Color.blue, "highlighted");
           }
+          return null;
         });
     CalendarPanel panel = new CalendarPanel(settings);
     settings.setVetoPolicy(
-        new DateVetoPolicy() {
-          @Override
-          public boolean isDateAllowed(LocalDate date) {
-            final int day = date.get(ChronoField.DAY_OF_MONTH);
-            return (day % 5 != 0);
-          }
+        date -> {
+          final int day = date.get(ChronoField.DAY_OF_MONTH);
```

Comment:
On a similar note as my other comment I would very much prefer the parameter name `dateToVeto` here,

Reply:
done